cluster/terraform_kubernetes/config/production.tfvars.json

```diff
@@ -46,7 +46,10 @@
     "SLACK_WEBHOOK_TV",
     "SLACK_WEBHOOK_CCBL",
     "SLACK_WEBHOOK_FALTRN",
-    "SLACK_WEBHOOK_GIT"
+    "SLACK_WEBHOOK_GIT",
+    "SLACK_WEBHOOK_IMS",
```

Comment:
I believe the short service name is ITTMS

Reply:
changed to ITTMS